regen_doc.g

```diff
@@ -1,5 +1,9 @@
 # Regenerate parts of the recog documentation from its source code
 
+if IsPackageLoaded("recog") then
+    ErrorNoReturn("another version of package recog is already loaded. ",
```

Comment:
of the package recog instead of package recog

Reply:
Thanks.

Reply:
I've changed the commit and removed this if block. :)